configure.ac

```diff
@@ -362,4 +375,10 @@ AC_CONFIG_FILES([
 	contrib/Makefile
 	])
 
+AC_CONFIG_FILES([sbin/firehol], [chmod +x sbin/firehol])
+AC_CONFIG_FILES([sbin/fireqos], [chmod +x sbin/fireqos])
+AC_CONFIG_FILES([sbin/link-balancer], [chmod +x sbin/link-balancer])
+AC_CONFIG_FILES([sbin/update-ipsets], [chmod +x sbin/update-ipsets])
+AC_CONFIG_FILES([sbin/vnetbuild], [chmod +x sbin/vnetbuild])
```

Comment:
Fine, now that that file exists, I think I finally get you. I must be getting slow.

Next update should be be closer still thanks for the continued feedback.